lti_consumer/plugin/views.py

```diff
@@ -835,4 +835,10 @@ def start_proctoring_assessment_endpoint(request):
         user_id=request.user.id,
     )
 
-    return render(request, 'html/lti_start_assessment.html', status=HTTP_200_OK)
+    context_url = "/".join([launch_data.context_title, "course", launch_data.context_id, launch_data.context_label])
```

Comment:
Question: I figure that this might be a clean/efficient way to concatenate the url but open to feedback here to make this better/more readable!

Reply:
I think this is readable. This is where you could grab the MFE URL via settings instead of using `context_tile`